src/quo2/components/buttons/button/view.cljs

```diff
@@ -93,46 +104,46 @@
            (when before
              [rn/view
               {:style (style/before-icon-style
-                       {:override-margins        override-before-margins
-                        :size                    size
-                        :icon-container-size     icon-container-size
-                        :icon-background-color   (get icon-background-color blur-state)
-                        :icon-container-rounded? icon-container-rounded?
-                        :icon-size               icon-size})}
+                       {:size                  size
+                        :icon-background-color (get icon-background-color blur-state)
+                        :icon-size             icon-size})}
               [quo2.icons/icon before
                {:color icon-secondary-color
                 :size  icon-size}]])
            [rn/view
             (cond
-              (or icon icon-no-color)
```

Comment:
there's no uses of `icon-no-color` -> removed it accordingly as it is not needed/ not in the designs